app.py

```diff
@@ -133,10 +134,12 @@ def _do_sync(
     if not only_one_kind or only_solver_documents:
         _LOGGER.info("Syncing solver results")
         category = "solver"
-        _METRIC_SECONDS.labels(category=category).time()
+        start = time.time()
```

Comment:
It would be better to use [monotonic time for this](https://docs.python.org/3/library/time.html#time.monotonic).

Reply:
Added thanks @fridex !